images/build/Dockerfiles/ltsVersions.Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG SDK_STORAGE_BASE_URL_VALUE
 # Determine where the image is getting built (DevOps agents or local)
 ARG AGENTBUILD
 
-FROM buildpack-deps:stretch AS main
+FROM githubrunners-buildpackdeps-stretch AS main
```

Comment:
This is so that the base layers are shared between the full build image and github-actions image in case of AppService scenario (since they are going to adopt dynamic install iteratively)